conda_smithy/lint_recipe.py

```diff
@@ -85,7 +85,7 @@ def lintify(meta, recipe_dir=None):
                     bad_selectors.append(selector_line)
         if bad_selectors:
             lints.append('Selectors are suggested to take a '
-                         '"  # [<selector>]" form.')
+                         '"<two spaces>#<one space>[<selector>]" form.')
```

Comment:
The failing test is because the specific string is tested in https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/tests/test_lint_recipe.py#L113.

Yay to tests! 😄